idprovider/local/idprovider_test.go

```diff
@@ -239,9 +239,9 @@ func Test_YAML_Delete_Read(t *testing.T) {
 func Test_YAML_UpdateStorage(t *testing.T) {
 	t.Run("happy_empty_file", func(t *testing.T) {
 		// Setup: NewYAML with zero entries
-		emptyFile := contactstest.NewYAMLFileT(t)
-		fileWithTwoPeers := contactstest.NewYAMLFileT(t, peer1, peer2)
-		c, err := contactsyaml.New(emptyFile, walletBackend)
+		emptyFile := idprovidertest.NewYAMLFileT(t)
+		fileWithTwoPeers := idprovidertest.NewYAMLFileT(t, peer1, peer2)
```

Comment:
```suggestion
		fileWithTwoPeers := idprovidertest.NewProviderT(t, peer1, peer2)
```

With this change, the ID provider type should only be referred to as Local and yaml or any other file format should be only an internal implementation detail.